test/conversion.test.js

```diff
@@ -27,6 +27,52 @@ describe('Curl converter should', function() {
 		});
 	});
 
+	describe('[Github #2] - set the method to ', function() {
+		it('GET if --get option is given in the curl command', function(done) {
+			convert({
+				type: 'string',
+				data: 'curl -d "key=example" --get https://example.com'
+			}, function (err, result) {
+				expect(result.result).to.equal(true);
+				expect(result.output[0].data.method).to.equal('GET');
+				done();
+			});
+		});
+
+		it('POST if only -d option is given in the curl command', function(done) {
+			convert({
+				type: 'string',
+				data: 'curl -d "key=example" https://example.com'
+			}, function (err, result) {
+				expect(result.result).to.equal(true);
+				expect(result.output[0].data.method).to.equal('POST');
+				done();
+			});
+		});
+
+		it('HEAD if --head or -I is given in the curl command', function(done) {
+			convert({
+				type: 'string',
+				data: 'curl --head https://example.com -d "a=b" --get'
```

Comment:
If we are checking for the hierarchy too, that even if --get method is provided, the --head takes the priority, we should mention it in the test name.